src/types/ScreenInfoUiaProviderBase.cpp

```diff
@@ -235,6 +235,10 @@ IFACEMETHODIMP ScreenInfoUiaProviderBase::GetSelection(_Outptr_result_maybenull_
     {
         return E_OUTOFMEMORY;
     }
+    else if (!_pData->IsUiaDataInitialized())

```

Comment:
This method is the only one where the lock is above these early returns - any good reason?